src/components/NavBar/style.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import { LinkColored } from '@/styles/text';
+
+export const NavBarContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 78px;
+  background: ${COLORS.blueMid};
+  z-index: 1000;
+  width: 100%;
+  box-shadow: 0px 4px 7px 0px rgba(0, 0, 0, 0.1);
+  justify-content: space-between;
+  padding: 15px 0px;
+  position: fixed;
```

Comment:
could we actually remove this line for now? We can update this in the second navbar sprint if we want it to be fixed